api/src/pcapi/core/finance/api.py

```diff
@@ -1618,9 +1618,9 @@ def get_collective_data(query: BaseQuery) -> BaseQuery:
 def _payment_details_row_formatter(sql_row: typing.Any) -> tuple:
     if hasattr(sql_row, "ministry"):
         booking_type = "EACC"
-    elif sql_row.deposit_type == models.DepositType.GRANT_15_17.value:
+    elif sql_row.made_by_underage_user == True:
         booking_type = "EACI"
-    elif sql_row.deposit_type == models.DepositType.GRANT_18.value:
+    elif sql_row.made_by_underage_user == False:
         booking_type = "PC"
     else:
         raise ValueError("Unknown booking type (not educational nor individual)")
```

Comment:
```suggestion
    else:
        booking_type = "PC"
```

tu peux aussi changer le `elif sql_row.made_by_underage_user == True` par `elif sql_row.made_by_underage_user`

Reply:
C'est très vrai, j'ai mécaniquement remplacé et oublié de rendre ça plus beau